### ISSUE TITLE
BUG: Initiate DTA attack button doesn't work

### ISSUE THREAD

**Issue by scooper4711** [toastygm/HarnMaster-3-FoundryVTT#141]
Version 0.7.18
<img width="283" alt="Screen Shot 2020-12-10 at 5 06 56 PM" src="https://user-images.githubusercontent.com/1970249/101844634-1f8d5580-3b0a-11eb-9093-50f1c16cc5be.png">

**Comment by scooper4711** [comment]
```
Uncaught (in promise) ReferenceError: isEquipped is not defined
    at combat.js:322
    at Array.filter (<anonymous>)
    at attackDialog (combat.js:322)
    at Module.meleeAttack (combat.js:204)
    at Module.weaponAttack (macros.js:446)
    at Function._onChatCardAction (actor.js:996)
    at HTMLElement.dispatch (jquery.min.js:2)
    at HTMLElement.v.handle (jquery.min.js:2)
```

**Comment by scooper4711** [comment]
fixed in 0.7.19
